### ISSUE TITLE
OutlineTextField and trailingIcon not clickable as expected

### ISSUE THREAD

**Issue by Dominaezzz** [JetBrains/compose-jb#308]
```
OutlinedTextField(
	value = "Content",
	onValueChange = { },
	label = { Text("Label") },
	trailingIcon = {
		IconButton(onClick = { println("WOAH!") }) {
			Icon(Icons.Filled.ArrowDropDown, null)
		}
	}
)
```

Clicking on the icon doesn't print `WOAH!`, in fact the button is disabled.
This was working fine on `0.3.0-build135` and broke with `0.0.0-vsync-build30`.

**Comment by Dominaezzz** [comment]
Appears to be fixed in version 0.3.0 .

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6460">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
